pkg/virt-controller/watch/migration_test.go

```diff
@@ -1562,29 +1553,24 @@ var _ = Describe("Migration watcher", func() {
 			addVirtualMachineInstance(vmi)
 			addNode(node)
 
-			shouldExpectGenericMigrationUpdate()
-
-			expectPodToHaveProperNodeSelector := func(pod *k8sv1.Pod) {
-				podHasCpuModeLabelSelector := false
-				for key, _ := range pod.Spec.NodeSelector {
-					if strings.Contains(key, virtv1.SupportedHostModelMigrationCPU) {
-						podHasCpuModeLabelSelector = true
-						break
-					}
-				}
-
-				Expect(podHasCpuModeLabelSelector).To(Equal(toDefineHostModelCPU))
-			}
-			kubeClient.Fake.PrependReactor("create", "pods", func(action testing.Action) (handled bool, obj k8sruntime.Object, err error) {
-				creation, ok := action.(testing.CreateAction)
-				Expect(ok).To(BeTrue())
-				pod := creation.GetObject().(*k8sv1.Pod)
-				expectPodToHaveProperNodeSelector(pod)
-				return true, creation.GetObject(), nil
-			})
 			controller.Execute()
 
 			testutils.ExpectEvent(recorder, SuccessfulCreatePodReason)
+			expectPodCreation(vmi.Namespace, vmi.UID, migration.UID, 1, 0, 1)
+			pods, err := kubeClient.CoreV1().Pods(vmi.Namespace).List(context.Background(), metav1.ListOptions{
+				LabelSelector: fmt.Sprintf("%s=%s,%s=%s", virtv1.MigrationJobLabel, string(migration.UID), virtv1.CreatedByLabel, string(vmi.UID)),
+			})
+			//TODO FIXME CONTAINS
```

Comment:
What do you mean?

Reply:
Leftover from implementing step.
Thank you, eagle eye :)